lewd/lewd_datums.dm

```diff
@@ -240,4 +260,6 @@
 	max_distance = 0
 
 /datum/interaction/lewd/grindmouth/display_interaction(mob/user, mob/target)
-	user.do_grindmouth(target)
\ No newline at end of file
+	user.do_grindmouth(target)
+
+/datum/inter
```

Comment:
`/datum/inter ` what is that doing there?

Reply:
Uhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh